client/my-sites/site-settings/disconnect-site/style.scss

```diff
@@ -1,34 +1,33 @@
+/** @format */
 .disconnect-site {
 	&.site-settings {
-	position: fixed;
-  top: 40%;
-  left: 50%;
-  transform: translate(-50%, -50%);
+		// Bit hacky -- compensate the padding added for the sidebar (which we aren't showing here).
+		margin-left: -340px;
```

Comment:
I've done a bit of Calypso archeology and I've got a "less-hacky" or at least "more-Calypso-ey" solution:

```patch
diff --git a/client/my-sites/site-settings/controller.js b/client/my-sites/site-settings/controller.js
index 7084a3000e..1376b14a33 100644
--- a/client/my-sites/site-settings/controller.js
+++ b/client/my-sites/site-settings/controller.js
@@ -26,6 +26,7 @@ import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/u
 import { isJetpackSite } from 'state/sites/selectors';
 import { canCurrentUser, isVipSite } from 'state/selectors';
 import { SITES_ONCE_CHANGED } from 'state/action-types';
+import { setSection } from 'state/ui/actions';
 
 function canDeleteSite( state, siteId ) {
 	const canManageOptions = canCurrentUser( state, siteId, 'manage_options' );
@@ -111,11 +112,13 @@ const controller = {
 
 	disconnectSite( context ) {
 		ReactDom.unmountComponentAtNode( document.getElementById( 'secondary' ) );
+		context.store.dispatch( setSection( null, { hasSidebar: false } ) );
 		renderPage( context, <DisconnectSite /> );
 	},
 
 	disconnectSiteConfirm( context ) {
 		ReactDom.unmountComponentAtNode( document.getElementById( 'secondary' ) );
+		context.store.dispatch( setSection( null, { hasSidebar: false } ) );
 		renderPage( context, <ConfirmDisconnection /> );
 	},
```

This sets the `has-no-sidebar` class on the reactroot, which takes care of the sidebar spacing. You can see it live by dispatching:

```js
dispatch( { type: 'SECTION_SET', hasSidebar: false });
```

Reply:
Hah, I knew there had got to be a better way! Thanks a million Jon!